Makefile

```diff
@@ -25,5 +25,5 @@ generate-api:
 	docker run --rm -v $(shell pwd):/app openapitools/openapi-generator-cli generate \
     -i ./app/openapi.json \
     -g typescript-axios \
-    --additional-properties=supportsES6=true,npmVersion=6.9.0,typescriptThreePlus=true,withSeparateModelsAndApi=true,modelPackage=models,apiPackage=domain,useSingleRequestParameter=true \
+    --additional-properties=supportsES6=true,npmVersion=6.9.0,typescriptThreePlus=true,withSeparateModelsAndApi=true,modelPackage=models,apiPackage=domain \
```

Comment:
Not using the single request param option opens up a different can of worms. 
The order of params in the function signature becomes significant which can cause breaking changes where there wouldn't have been previously if the order of params changed or if say we have a param that was optional but is changed to be required, that would mean all users will have to reorder their function call params to the correct order.